hack/tools/Makefile

```diff
@@ -82,6 +83,10 @@ kustomize: $(KUSTOMIZE) $(SRCS) ## Build kustomize
 $(KUSTOMIZE): go.mod
 	go build -tags=tools -o $@ sigs.k8s.io/kustomize/kustomize/v4
 
+conversion-verifier: $(CONVERSION_VERIFIER) $(SRCS) ## fetch CAPI's conversion verifier
+$(CONVERSION_VERIFIER): go.mod
```

Comment:
@srm09 - I have added the new make target here to generate the conversion-verifier binary